api/proto/config/netconfig.proto

```diff
@@ -44,3 +47,22 @@ message NetworkAdapter {
         // firewall
         repeated ACE acls = 40;
 }
+
+message WirelessConfig {
+    WirelessType type = 1;            // either LTE or Wifi
+    repeated string CellularAPN = 5;  // APN for Cellular LTE, etc
```

Comment:
Nit; should it be cellularAPN to follow the same capitalization?

Reply:
Would it make sense to define a CellularConfig message to make it easier to add any future fields should we need more than the APN down the road?

Reply:
ok.
sure can put in a structure for the celluar.